utils/preprocessETH.py

```diff
@@ -0,0 +1,121 @@
+import pandas as pd
+import argparse
+import matplotlib.pyplot as plt
+import os
+import glob
+import numpy as np
+from myparser import getYamlConfig
+def unixtime(df, init_time = 1694563200.0):
+    #function that maps frame_ID to unixtime
```

Comment:
Please follow PEP8 to properly comment all functions in code.

https://peps.python.org/pep-0008/#documentation-strings